show-device-location-using-indoor-positioning/src/main/java/com/esri/arcgismaps/sample/showdevicelocationusingindoorpositioning/MainActivity.kt

```diff
@@ -142,10 +142,8 @@ class MainActivity : AppCompatActivity() {
             // load each FeatureTable
             featureTable.load().onSuccess {
                 // IPS_Positioning table needs to be present
-                if (featureTable.tableName == "IPS_Positioning") {
+                if (featureTable.tableName == "ips_positioning") {
```

Comment:
umm, this is not true for the building M dataset(I used for testing purpose) that works with the sample without any code change